### PR TITLE
React capstone project 

### PR DESCRIPTION
**Pull request by Rocio01**
Greetings 

For this capstone, I created a pokemon (SPA).

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.

Thanks for the attention.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Rocio01 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "sourceType": "module"
+  },
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",  
+    "linebreak-style": 0
```
Comment (mikenath223) [inline .eslintrc.json:22]: - [ ] Kindly use the exact eslint rules stated [here](https://github.com/microverseinc/linters-config/tree/master/react-redux).

File: src/tests/PokemonsContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import '@testing-library/jest-dom/extend-expect';
+import PokemonsContainer from '../components/PokemonsContainer';
+import store from '../redux/configureStore';
+
+describe('pokemons container component', () => {
+  test('matches PokemonsContainer component snapshot', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
```
Comment (mikenath223) [inline src/tests/PokemonsContainer.test.js:11]: - [ ] There are 2 failing test cases, kindly check and fix them.

Comment (rloterh) [inline src/tests/PokemonsContainer.test.js:11]: - [ ] Some of your test cases are yet failing. Kindly ensure to fix the same.

**Failing test case:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Rocio, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Kindly take your time to improve upon those issues outlined in the code review(s), and please check to see that the changes are working as intended._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 3 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rocio01** [comment]
Greetings 
Thanks for the review, I delete the repo and cloned it again on two different computers, and the tests are passing.
Kind regards

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**